source/octod/core.d

```diff
@@ -217,6 +217,15 @@ struct HTTPConnection
                 continue;
             }
 
+            // Most responses are JSON, but there are also some plain
+            // text ones that have to be handled differently:
+
+            if (accept == ResponseFormat.Version)
+            {
+                import vibe.stream.operations;
+                return Json(response.bodyReader.readAllUTF8());
+            }
```

Comment:
I think this is a bit more complex, there are other requests that are not json. Related to the design of how `accept` should be constructed.